daq_macros.py

```diff
@@ -1879,9 +1882,17 @@ def snakeRasterBluesky(rasterReqID, grain=""):
     logger.debug(f'lastOnSample(): {lastOnSample()} autoRasterFlag: {autoRasterFlag}')
     if (lastOnSample() and not autoRasterFlag):
       govStatus = gov_lib.setGovRobot(gov_robot, 'SA', wait=False)
+      if govStatus.exception():
+        logger.error(f"Problem during final governor move, aborting! exception: {govStatus.exception()}")
+        return
+
       targetGovState = 'SA'
     else:
       govStatus = gov_lib.setGovRobot(gov_robot, 'DI')
+      if govStatus.exception():
+        logger.error(f"Problem during final governor move, aborting! exception: {govStatus.exception()}")
```

Comment:
is end-of-collection ok?

Reply:
Probably I'm misreading the code? This is in the else block for "if (lastOnSample() and not autoRasterFlag):" so I was guessing that it wasn't the final move in this collection - if I'm wrong, please ignore!

Reply:
well, it is the last governor move of this block/data collection. it may not be the last one if there are more samples and in an automated mode. that is why I think end-of-collection is better (over final)